active/website.md

```diff
@@ -0,0 +1,63 @@
+# Website Working Group
+
+## Scope of responsibilities
+
+This is a replacement of the current maintainer team @django/djangoproject-com-maintainters. The team will own maintenance of the website codebase, and liaise with the @django/ops-team for production infrastructure considerations.
+
+The duties of the working group are:
+- Introduce new features on the website
+- Maintain and monitor the website 
+- Ensure that information on the website is accurate 
+- Help to make the website accessible to all
+
+
+### Delegated responsibilities:
+- Chair, Co-chair and Board Liaison can make final decision of  sections of the website that might not conform with DSF's CoC.
+- Members can work on bugs
```

Comment:
```suggestion
```

I believe anyone can contribute to the site as far as working on bugs, so we shouldn’t list this here unless we were intending to remove this option from people outside the group.

Reply:
For me the key wording is **can** meaning it's not only group members that can contribute.

Reply:
@ronnzw yes, agreed, though I still think my suggestion stands. This section is about "Delegated responsibilities", which are things that the DSF _Board_ delegates to the group. So though fixing bugs can be done by this group, it’d be more meaningful for the charter to focus on things that are bigger responsibilities.

Reply:
@thibaudcolas makes sense.